frontend/src/static/css/v_authorship.scss

```diff
@@ -2,6 +2,14 @@
 
 /* Authorship */
 #tab-authorship {
+
+  .file-content {
+    background-color: mui-color('github', 'background-color');
```

Comment:
Github's background colour for code is actually white (#fff), only the title is (#fafbfc)

Reply:
![Screen Shot 2020-03-03 at 1 06 02 AM](https://user-images.githubusercontent.com/35946746/75699470-7cdab780-5ceb-11ea-8bd8-6444a2342637.png)

So as seen, its different colour on the left. I have tried to go for that. 

Currently, the not shown code is shown as `fafbfc` but when you open the code it will be white. If we wish to go to full Github style, maybe we can make the not shown code, blue and then white when you open it, like github. What do you suggest ?

Reply:
We can just follow GitHub right? Meaning when its untouched code that is initially shown, it should be white background. If its untouched code that is requires toggling to be shown, then it should be `fafbfc` background